src/components/cells/cell-editor.jsx

```diff
@@ -15,12 +15,23 @@ import 'codemirror/addon/display/autorefresh'
 import 'codemirror/addon/comment/comment'
 import 'codemirror/addon/hint/show-hint'
 import 'codemirror/addon/hint/javascript-hint'
+import 'codemirror/addon/mode/simple'
 import '../../codemirror-keymap-sublime'
 
 import { getCellById } from '../../tools/notebook-utils'
 import * as actions from '../../actions/actions'
 import { postMessageToEvalFrame } from '../../port-to-eval-frame'
 
+CodeMirror.defineSimpleMode('fetch', {
```

Comment:
this is great! let's move it to a separate file tho

Reply:
yeah, was just thinking this earlier - will do.